container/factory/factory.go

```diff
@@ -21,6 +22,8 @@ func NewContainerManager(forType instance.ContainerType, conf container.ManagerC
 	switch forType {
 	case instance.LXC:
 		return lxc.NewContainerManager(conf, imageURLGetter, looputil.NewLoopDeviceManager())
+	case instance.LXD:
```

Comment:
I would think we need similar support for LoopDeviceManager from LXC containers. But I suppose that can be on a TODO.